qiskit/transpiler/passes/layout/vf2_post_layout.py

```diff
@@ -374,26 +373,4 @@ def _score_layout(self, layout, bit_map, reverse_bit_map, im_graph):
                         props = self.target[gate][qargs]
                         if props is not None and props.error is not None:
                             fidelity *= (1 - props.error) ** count
-        else:
```

Comment:
How was that working with `self.properties` already being removed? :-)

Reply:
It was essentially raising a `BackendPropertyError` and always passing... thanks to this removal I realized that oversight.